services/blockchain-indexer/shared/dataService/business/transactionsEstimateFees.js

```diff
@@ -244,6 +246,36 @@ const calcAdditionalFees = async (transaction) => {
 	return additionalFees;
 };
 
+const getTransactionParamsSchema = (transaction, metadata) => {
+	const moduleMetadata = metadata.modules.find(m => m.name === transaction.module);
+	const { params: schema } = moduleMetadata.commands.find(c => c.name === transaction.command);
+	return schema;
+};
+
+const validateTransactionParams = async transaction => {
+	// Mock optional values if not present before schema validation.
+	if (transaction.module === MODULE.INTEROPERABILITY
```

Comment:
This should be token module
```suggestion
	if (transaction.module === MODULE.TOKEN
```